src/test/java/racing/CarTest.java

```diff
@@ -0,0 +1,40 @@
+package racing;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author han
+ */
+class CarTest {
+
+    Car car;
+
+    @BeforeEach
+    void setUp() {
+        car = new Car(0);
```

Comment:
Car 객체를 생성할때마다 `0`을 항상 넣어줘야 하는데요,
정적 팩토리 메서드를 사용하면 의미있는 생성자를 제공할 수 있어요.
```suggestion
        car = Car.create();
```

이펙티브 자바의 아이템 1번을 참고해보시면 좋을 것 같아요.